satellitevu/apis/otm.py

```diff
@@ -57,7 +61,12 @@ def post_feasibility(
         response = self.make_request(
             method="POST", url=url, json={k: v for k, v in payload.items() if v}
         )
-        return response.json()
+
+        if response.status != 202:
+            raise Exception(f"Error - {response.status} : {response.text}")
+
+        response_bytes = raw_response_to_bytes(response)
+        return loads(response_bytes.read())
```

Comment:
What's the difference between jumping through these hoops and just `response.json()`?

Reply:
I was having a JSON decoder problem but it seems that was symptomatic of an error that I must have fixed elsewhere